mParticle-Singular/MPKitSingular.m

```diff
@@ -0,0 +1,344 @@
+//
+//  MPKitSingular.m
+//
+//  Copyright 2016 mParticle, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "MPKitSingular.h"
+#import "Singular.h"
+
+/* Import your header file here
+ */
+//#if defined(__has_include) && __has_include(<CompanyName/CompanyName.h>)
+//#import <CompanyName/CompanyName.h>
+//#else
+//#import "CompanyName.h"
+//#endif
+
+// This is temporary to allow compilation (will be provided by core SDK)
+NSUInteger MPKitInstanceCompanyName = 119;
```

Comment:
please update this to be Singular, and also remove all of the boiler plate example code from this file.